config.py

```diff
@@ -109,16 +106,13 @@ class DevelopmentConfig(Config):
     SECURITY_USER_NAME = os.getenv('SECURITY_USER_NAME', 'admin')
     SECURITY_USER_PASSWORD = os.getenv('SECURITY_USER_PASSWORD', 'secret')
     BASIC_AUTH = (SECURITY_USER_NAME, SECURITY_USER_PASSWORD)
-    DJANGO_CLIENT_ID = os.getenv('DJANGO_CLIENT_ID', 'ons@ons.gov')
-    DJANGO_CLIENT_SECRET = os.getenv('DJANGO_CLIENT_SECRET', 'password')
-    DJANGO_BASIC_AUTH = (DJANGO_CLIENT_ID, DJANGO_CLIENT_SECRET)
-    JWT_SECRET = os.getenv('JWT_SECRET', 'testsecret')
     USERNAME = os.getenv('RAS_BACKSTAGE_USERNAME', 'user')
     PASSWORD = os.getenv('RAS_BACKSTAGE_PASSWORD', 'pass')
     USE_UAA = int(os.getenv('USE_UAA', '1'))
-    UAA_SERVICE_URL = os.getenv('UAA_SERVICE_URL', 'localhost')
+    UAA_SERVICE_URL = os.getenv('UAA_SERVICE_URL', 'http://localhost')
```

Comment:
This looks odd to me, uaa would be at localhost without a port?  Not a huge deal, just a thought I had while looking

Reply:
defaults to 80?

Reply:
Added port